examples/vision/siamese.py

```diff
@@ -0,0 +1,247 @@
OKLAII3vtab2UIgegJnC953333XcuTIYX/++ad3LGHZpk0bP582bZpdf/31EfE5b948F6Ndu3a1PHny2Lfffmtz5syxyy+/3LZu3eptPvzwQ7vuuuvcEvvWW2/5c71797aRI0faTTfd5Bbi82cwduxYmzVrlt9++umn3fIcPCMXhiJFinjb4B1BHb8QgAAEIBA3BBC8cbPUTBQCyUTgbMGrDWw33nhjxMe2ePHiEdcDuREUKFDADh48aBKvEqzDhw930as2cjuYPHmyycL722+/+eg++ugjF8my3L755pve5vXXX7cRI0aYxPCKFSvOmcWkSZOsSpUqbhnu1KmTnz/zzDP+jDawFSxYEMvuOcS4gAAEIBCXBBC8cbnsTBoCURCQBVcWWhX5315xxRUuanUtUSsrq8r48eMtb9687rpwzz33eLQE3Zd4lRvE8uXLbcOGDZY+fXobMGCA/fTTT9alSxdvs2PHDq/PkCGD/fDDD96fIjGozdkb2mQlrl+/vq1bt8769Onj7RWNQc/IhSFXrly2dOlS27hxo33yySemfikQgAAEIBB3BBC8cbfkTBgCURKQJVWWXBVZWPPly2eKvnDmzBkrWbKkvfDCC143ZcoUdyeQu4MsuRLJ2tjWv39/e+SRR+zuu+/2qAlyfShdurS7Qrz44oveZvXq1d6HIixoQ9udd97pz5xv4ZWVOLDoDhs2LBKS7Oeff/axVKxY0Te8aQOd/Itnz57t/fIHAhCAAATiigCCN66Wm8lCIEoCChG2e/dut5Tq/K+//nILrs5lVVUs3CD0lzaryVVBQlhl7dq1tnjxYj/Xs8uWLfO+Vq1aZTpUFJVBG9Vk+Q2KIjbIunv8+PHgVuRX7hL79+/36z/++MNOnjzp5wpjpg1yGs+vv/7qh9wkjhw5EmnLCQQgAAEIxA0BBG/cLDUThUAICUgw33vvvVa+fHl3hZDPb/fu3UM4UoYEAQhAAAJpmACCNw0vHkOHQEwQkOVWkRbkmqCNaRQIQAACEIBAMhNA8CYzULqDAAQgAAEIQAACEAgXAQRvuNaD0UAAAhCAAAQgAAEIJDMBBG8yA6U7CMQUgWPHjkU2fClxgw6FCUut8F6K3/vxxx87U6Uvfu2112KKL5OBAAQgAIFUIYDgTRXMvAQCaZTAwoULPeyYUgQrta9+FUWhYcOGqTKja665xlq2bOnvUqxeBG+qYOclEIAABGKNAII31laU+UAgOQko9W+mTJls6NCh9uOPP/qhJA5Bml6FFlPMW1l8lRlNiSDOLkHGtAULFpx92xNVKAGFBKwSTpxdvvzySxs0aJDff+CBByJRG/T+4L0rV660LVu22L59+3xsanN2mT9/vo9HYckU8uzsMGfTp0/3JBWLFi06uwnnEIAABCAQuwQQvLG7tswMAtETmDdvnmXPnt3mzp2baGfVq1e3EiVKeLIJWYCzZs0aibUrYVmkSBHPeKZUwg0aNPCYvBK4xYoV81BkNWvWtBtuuME++OAD718JKmRBrlGjhj311FNuVX7uuee8rkKFCnbffff5uSzMSn6hRBX6VUY2iVyVIUOG+LWe7dChg+XMmdNeffVVf3ezZs08/XDXrl0tf/78Xq8YwhQIQAACEIhpAgjemF5eJgeBKAnIpUHCsEyZMqb0wIqXe/PNN3vmNHUt4SnBGiSbUGphZVo7cOCA5c6d25544gkfgSyyTz75pCeqePTRR73PYGh169b1/tWmcuXKJquuipJWyKVB6YZVJGCVRlilbdu2duWVV7p1WdfKpNa+fXs7ceKEKZbv2e+VCFcKZCW7yJgxozVv3tz70Ji/+eYbP+cPBCAAAQjENAEEb0wvL5ODQJQEZDVVSt5u3bqZUgUrlbA2kimlr4ostBKsQalVq5Y9++yzpuxol156qc2cOTOoivxKkEr8BmX48OFWsGBBU9pgWYtljQ2KRHbr1q39UkJYKYlVGjdubLIuB6VJkybWqlUr27lzp6cwHjdunFedOnXK+2PPtzEAAALKSURBVHzllVf8WuPX+3U0atTIXTSCPviFAAQgAIGYJYDgjdmlZWIQSAYCcmXIkiVLJPXv+V3K4lqvXr3I7SpVqngCCaX7lYVXfr0qSumrtMFKBSwRK6EcFCWc0Ga4bdu2Wbly5dxSG9TJTUKWWxUJ3mCznCy4gXuD6urUqeMiWv3nzZvXXnrpJW8jH16NI3Bp8JtmJouzxn711VenWsSJ4N38QgACEIBAqhNA8KY6cl4IgTREYNasWZYuXTqrXbu2uyr06NHDOnbs6Ju+5CIgv1yJ1aDIpeHhhx/2S/nJSmwqi5pErtwTDh06ZO+//75bf/v3729vv/22++xK9Kqob7kdjB492kWq3h1EabjttttMfrwqckvIkyePBf63t956q1WrVs3r2rRpY5kzZ/bNbnKHkMvF+PHj3Z2iYsWK1rt3b7dQ67l8+fLZnj17vB1/IAABCEAgZgkgeGN2aZkYBJKBgKIbyEVBQlSuBTrkPqANZXIXmDp1qgXuAnqdYuUG7gSnT5+2nj17mkSmBGrgBqHnZO2VQJVFWBEZAuH6xx9/uH+uBLJEqjagJSQk+EwGDx5scn9QkXuFrLhBu9dffz1Sp/phw4a54F2yZImL5F69enk7uVhIkFetWtVatGjhrhdewR8IQAACEIhlAgjeWF5d5gaBeCSgDWp9+/b1qS9dutQt1BLPZxdZpykQgAAEIBA3BBC8cbPUTBQCcUJg9uzZVrp0abfsli1b1q3OgSU4ThAwTQhAAAIQOJcAgvdcHlxBAAKxQkDuGLt27YqV6TAPCEAAAhBIOgEEb9LZ0RICEIAABCAAAQhAIA0QQPCmgUViiBCAAAQgAAEIQAACSSeA4E06O1pCAAIQgAAEIAABCKQBAi5408A4GSIEIAABCEAAAhCAAASSRiCdme0ws784YMA3wDfAN8A3wDfAN8A3wDcQg9/Ajv8DonQ+dvW5neEAAAAASUVORK5CYII=)
```

Comment:
There are over 44.7 thousand characters here. Are you sure this is a correct image link?

Reply:
It is bas64 encoded image itself, renders automaticaaly in webpages and notebooks. 
But I guess I should remove it.